images/osm2pgsql/run.py

```diff
@@ -39,9 +44,65 @@ def pg_is_ready():
             log.info('pg is ready for connections')
     return ready
 
+
+def first_import():
+    log.info('starting first osm import')
+    os.system("osm2pgsql \
+                --create \
+                --slim \
+                -G \
+                --hstore \
+                --tag-transform-script /src/openstreetmap-carto.lua \
+                -C 2500 \
+                --number-processes 4 \
+                -S /src/openstreetmap-carto.style \
+                /src/first-osm-import.osm"
+              )
+    log.info('first import is done')
+
+
+def parse_integer_to_directory_number(integer):
+    return "{0:0=3d}".format(integer)
+
+
+def is_file_or_directory(path):
+    return os.path.exists(path)
+
+
+def get_file_from_replication_server(path):
+    url = '/'.join([REPLICATION_URL, path])
```

Comment:
I think `'{0}/{1}'.format(REPLICATION_URL, path)` is more readable, wdyt?

Reply:
I don't mind any of the solutions in terms of readability. i think using join might be more efficient but it really doesn't matter in this context (also should've used tuple instead of list anyway as its immutable).
up to you to decide.

Reply:
I prefer `'{0}/{1}'.format(REPLICATION_URL, path)`